neurons/validator.py

```diff
@@ -972,16 +997,18 @@ def log_step(
         table.add_column("win_rate", style="magenta")
         table.add_column("win_total", style="magenta")
         table.add_column("weights", style="magenta")
+        table.add_column("competition weights", style="magenta")
         table.add_column("block", style="magenta")
         table.add_column("competition", style="magenta")
-        for uid in uids:
+        for idx, uid in enumerate(uids):
             try:
                 table.add_row(
                     str(uid),
                     str(round(step_log["uid_data"][str(uid)]["average_loss"], 4)),
                     str(round(step_log["uid_data"][str(uid)]["win_rate"], 4)),
                     str(step_log["uid_data"][str(uid)]["win_total"]),
                     str(round(self.weights[uid].item(), 4)),
```

Comment:
do weights and sub competition weights not get indexed into the same way? the uid should be fine without the enumerate?

Reply:
That's right
Done